en/docs/api-analytics/cloud-analytics/elk-installation-guide.md

```diff
@@ -0,0 +1,298 @@
+
+2. Make sure Elasticsearch is [up and running](https://www.elastic.co/guide/en/elastic-stack-get-started/current/get-started-elastic-stack.html#_make_sure_that_elasticsearch_is_up_and_running).
+
+!!! info
+    As recommended by ELK, a minimum 3 node cluster is required for a production environment.
+
+
+#### Installing Filebeat
+
+1. [Install Filebeat](https://www.elastic.co/guide/en/beats/filebeat/7.13/filebeat-installation-configuration.html#installation) according to your operating system.
+
+2. Configure **Filebeats** to read the log file in the `repository/logs` folder. 
+
+    ```
+    filebeat.inputs:
+    - type: log
+    enabled: true
+    paths:
+        - {apim_home}/repository/logs/apim_metrics.log
+    include_lines: ['(apimMetrics):']
+    output.logstash:
+    # The Logstash hosts
+    hosts: ["{LOGSTASH_URL}:5044"]
+    ```
+
+#### Installing Logstash
+
+1. [Install Logstash](https://www.elastic.co/guide/en/logstash/current/installing-logstash.html) according to your operating system.
+
+    ```
+    input {
+    beats {
+        port => 5044
+    }
+    }
+
+    filter {
+    grok {match => ["message", "%{GREEDYDATA:UNWANTED}\ apimMetrics:%{GREEDYDATA:apimMetrics}\, %{GREEDYDATA:UNWANTED} \:%{GREEDYDATA:properties}"]}
+    json {source => "properties"}
+    }
+    output {
+    if[apimMetrics] == " apim:response" {
+    elasticsearch {
+        hosts => ["http://{ELK_URL}:9200"]
+        index => "apim_event_response"
+        user => "elastic"
+        password => "Admin1234"
+    }
+    } else if[apimMetrics] == " apim:faulty" {
+    elasticsearch {
+        hosts => ["http://{ELK_URL}:9200"]
+        index => "apim_event_faulty"
+        user => "elastic"
+        password => "Admin1234"
+    }
+    }
+    }
+    ```
+
+#### Installing Kibana
+
+1. [Install Kibana](https://www.elastic.co/guide/en/elastic-stack-get-started/current/get-started-elastic-stack.html#install-kibana) according to your operating system.
+
+2. [Launch](https://www.elastic.co/guide/en/elastic-stack-get-started/current/get-started-elastic-stack.html#_access_the_kibana_web_interface) the Kibana web interface.
+
+3. Log in to the Kibana dashboards.
+
+4. Navigate to Stack Management > index pattern. If you already have any index patterns created under the following names, delete them before importing the saved artifacts.
+    ```
+    apim_event*
+    apim_event_faulty
+    apim_event_response
+    ```
+
+5. Download the artifact file [here]({{base_path}}/assets/img/analytics/cloud/export.ndjson).
+
+6. Navigate to **Stack Management** > **Saved Object** and click on **Import**. Add the downloaded artifact file as an import object, and import
+
+!!! info
+    Follow the recommendations of Elastic in order to optimize the performance of the system.
+
+
+### Step 3 - Configure Security in ELK
```

Comment:
This topic can be the Step 3, and inside of this,

-  Configure Basic Authentication
-  Configure Single-Sign-On with WSO2 API Manager via OpenID Connect
has to be subtopics.

Reply:
Added a note mentioning that either of these 2 steps can be followed.